BrowserKit/Sources/Shared/UserAgent.swift

```diff
@@ -102,6 +102,7 @@ struct CustomUserAgentConstant {
     private static let safariMobileUA = UserAgentBuilder.defaultMobileUserAgent().clone(extensions: "Version/18.6 \(UserAgent.uaBitMobile) \(UserAgent.uaBitSafari)")
 
     static let customMobileUAForDomain = [
+        "tver.jp": safariMobileUA,
         "epic.com": safariMobileUA,
```

Comment:
@denschub Sooo… following the pattern from previous shims, this only adds the override to the mobile default, or precisely "show mobile site" mode. It can e.g. break on mobile if specifically requested "show desktop site" (I'm currently investigating why the `FxiOS` bit is kept in that request), but OTOH the iPadOS default will be spoofing Safari anyway so its default "show desktop site"-like behavior out of the box shouldn't have issues with that.

Reply:
That sounds good to me. Thanks for your work and the context here!

Reply:
(I'll be following up about the `platform: .Mobile` vs. `platform: .Desktop` difference between iPhone and iPad separately, mostly through #25065 before spinning it out from that once I gather more uastrings… and resolution of that should eventually help even out also the wrinkle mentioned above.)